exercises/phone-number/phone_number_test.go

```diff
@@ -20,6 +22,9 @@ var numberTests = []testCase{
 }
 
 func TestNumber(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, wanted %v.", testVersion, targetTestVersion)
+	}
 	for _, test := range numberTests {
 		actual, actualErr := Number(test.input)
 		if actual != test.expected {
```

Comment:
And also do the same for `testFormat` and `testAreaCode` except I can't put the comments there since they are too many lines away from a changed line